apps/addons/tests/test_models.py

```diff
@@ -2294,6 +2294,65 @@ def test_enable_addon(self, file_mock):
         assert not mock.hide_disabled_file.called
 
 
+class TestAddonWatchDeveloperNotes(amo.tests.TestCase):
+
+    def make_addon(self, **kwargs):
+        addon = Addon(type=amo.ADDON_EXTENSION, status=amo.STATUS_PUBLIC,
+                      **kwargs)
+        addon.save()
+        addon.versions.create(has_info_request=True)
+        addon.versions.create(has_info_request=False)
+        addon.versions.create(has_info_request=True)
+        return addon
+
+    def assertHasInfoSet(self, addon):
+        assert any([v.has_info_request for v in addon.versions.all()])
+
+    def assertHasInfoNotSet(self, addon):
+        assert all([not v.has_info_request for v in addon.versions.all()])
+
+    def test_has_info_save(self):
```

Comment:
Docstring please (something like `Saving an add-on without changing whiteboard or adding comments doesn't change the has_info_request`)